Projects/MoveFlow/Sources/View/MovingFlowAddExtraBuildingView.swift

```diff
@@ -0,0 +1,170 @@
+import Flow
+import SwiftUI
+import hCore
+import hCoreUI
+import hGraphQL
+
+struct MovingFlowAddExtraBuildingView: View {
+    @StateObject var vm = MovingFlowAddExtraBuildingViewModel()
+    var body: some View {
+        hForm {
+            VStack(spacing: 8) {
+                hSection {
+                    typeOfBuilding
+                }
+                .sectionContainerStyle(.transparent)
+                hSection {
+                    livingArea
+                }
+                .sectionContainerStyle(.transparent)
+                hSection {
+                    connectedToWater
+                }
+                hSection {
+                    VStack {
+                        hButton.LargeButton(type: .primary) {
+                            withAnimation {
+                                vm.addExtraBuilding()
+                            }
+                        } content: {
+                            hText(L10n.generalSaveButton)
+                        }
+                        hButton.LargeButton(type: .ghost) {
+                            vm.dissmisAddExtraBuilding()
+                        } content: {
+                            hText(L10n.generalCancelButton)
+                        }
+
+                    }
+                    .padding(.vertical, 16)
+                }
+                .sectionContainerStyle(.transparent)
+            }
+        }
+        .hDisableScroll
+    }
+    @ViewBuilder
+    private var typeOfBuilding: some View {
+        hFloatingField(
+            value: vm.buildingType?.translatedValue ?? "",
+            placeholder: L10n.changeAddressExtraBuildingContainerTitle,
+            error: $vm.buildingTypeError
+        ) {
+            vm.showTypeOfBuilding()
+        }
+    }
+
+    private var livingArea: some View {
+        hFloatingTextField(
```

Comment:
Don't we want m2 placeholder for this one as well?

Reply:
William was all for removing this m2